package.json

```diff
@@ -39,6 +39,7 @@
     "liftoff": "3.1.0",
     "lodash": "^4.17.15",
     "mkdirp": "^0.5.1",
+    "p-timeout": "^3.2.0",
```

Comment:
Considering how small p-yimeout is, maybe it would make sense to inline implementation and use native finally instead of p-finally?

Reply:
no strong opinion on that, though, sindresorhus libs are legit.

Reply:
Done